<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stern_attack_view</class>
 <widget class="QWidget" name="stern_attack_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_state">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="6">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLabel" name="label_step1">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QLabel" name="label_step2">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6" colspan="2">
    <widget class="QLabel" name="label_step3">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="5">
    <widget class="QScrollArea" name="scrollArea_currstate">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>127</width>
        <height>375</height>
       </rect>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit_state">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>131</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QScrollArea" name="scrollArea_step1">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>126</width>
        <height>155</height>
       </rect>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit_step1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>131</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QScrollArea" name="scrollArea_step2">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>127</width>
        <height>155</height>
       </rect>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit_step2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>131</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QScrollArea" name="scrollArea_step3">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>126</width>
        <height>155</height>
       </rect>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit_step3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>131</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLabel" name="label_step4">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QLabel" name="label_step5">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6" colspan="2">
    <widget class="QLabel" name="label_step6">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QScrollArea" name="scrollArea_step4">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>126</width>
        <height>155</height>
       </rect>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit_step4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>131</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QScrollArea" name="scrollArea_step5">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>127</width>
        <height>155</height>
       </rect>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit_step5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>131</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QScrollArea" name="scrollArea_step6">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>126</width>
        <height>155</height>
       </rect>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit_step6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>131</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="4" column="2" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QPushButton" name="pushButton_quit">
     <property name="text">
      <string>QUIT</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="pushButton_saveresult">
     <property name="text">
      <string>TRY 1000 TIMES</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButton_tryagain">
     <property name="text">
      <string>TRY AGAIN</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
